nms/app/packages/magmalte/docker-compose-e2e.yml

```diff
@@ -0,0 +1,85 @@
+# Provides all dependent services
```

Comment:
I think the only difference here is the addition of the `mock_server`. Can we instead just use the provided docker-compose file and then just `docker run mock_server ...` to get the mock server up? This way we also get to test the compose file while we're at it

Reply:
one other reason is that I am running magmalte container 
    ```command: "/usr/local/bin/wait-for-it.sh -s -t 30 mariadb:3306 -- yarn run start:prod"``` instead of 
`yarn run start:dev`. The reason I'm doing that is because of some weird issues I ran into CI when i did a docker run. 
Like the one pasted below.
```
magmalte_1     | Error from chokidar (/usr/src/node_modules/fbt/lib): Error: ENOSPC: System limit for number of file watchers reached, watch '/usr/src/node_modules/fbt/lib/IntlCLDRNumberType24.js'
magmalte_1     | Error from chokidar (/usr/src/node_modules/fbt/lib): Error: ENOSPC: System limit for number of file watchers reached, watch '/usr/src/node_modules/fbt/lib/IntlCLDRNumberType24.js.flow'
magmalte_1     | Error from chokidar (/usr/src/node_modules/fbt/lib): Error: ENOSPC: System limit for number of file watchers reached, watch '/usr/src/node_modules/fbt/lib/IntlCLDRNumberType25.js'
magmalte_1     | Error from chokidar (/usr/src/node_modules/fbt/lib): Error: ENOSPC: System limit for number of file watchers reached, watch '/usr/src/node_modules/fbt/lib/IntlCLDRNumberType25.js.flow'
magmalte_1     | Error from chokidar (/usr/src/node_modules/fbt/lib): Error: ENOSPC: System limit for number of file watchers reached, watch '/usr/src/node_modules/fbt/lib/IntlCLDRNumberType26.js'
magmalte_1     | Error from chokidar (/usr/src/node_modules/fbt/lib): Error: ENOSPC: System limit for number of file watchers reached, watch '/usr/src/node_modules/fbt/lib/IntlCLDRNumberType26.js.flow'
magmalte_1     | Error from chokidar (/usr/src/node_modules/fbt/lib): Error: ENOSPC: System limit for number of file watchers reached, watch '/usr/src/node_modules/fbt/lib/IntlCLDRNumberType27.js'
magmalte_1     |CERROR: Aborting.
```